.github/workflows/test-suite-desktop-e2e-fw-canary.yml

```
@@ -56,6 +56,7 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v4
         with:
+          submodules: true
```

actionlint (1.7.4)

input "submodules" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)